packages/atomic/src/components/common/item-list/display-grid.tsx

```diff
@@ -12,35 +11,22 @@ export interface DisplayGridProps {
 }
 
 export const DisplayGrid: FunctionalComponent<DisplayGridProps> = (
-  {item, setRef, select, gridTarget, beginDelayedSelect, cancelPendingSelect},
+  {setRef},
   children
 ) => {
+  let ref: HTMLElement | undefined;
   return (
     <div
       part="result-list-grid-clickable-container outline"
-      ref={(element) => setRef(element)}
+      ref={(element) => {
+        ref = element;
+        setRef(element);
+      }}
       onClick={(event) => {
         event.preventDefault();
-        select();
-        window.open(
-          item.clickUri,
-          event.ctrlKey || event.metaKey ? '_blank' : gridTarget,
-          'noopener'
-        );
+        ref?.querySelector('atomic-result')?.click();
       }}
```

Comment:
Almost useless: but if you put some fat margin/padding, and click in the space between the border of the div and the actual result component, well, your click fell thru.